devlab/config.template

```diff
@@ -113,6 +114,7 @@ ext_cidr = <dst_ext_cidr>
 user = <dst_user>
 password = <dst_password> 
 tenant = <dst_tenant>
+service_tenant = <dst_name_of_tenant_for_the_openstack_services>
```

Comment:
Please put actual value which is `service` for devlab. I don't think we need it configurable right now.